Assets/Scripts/ECS/Death/DeathCause.cs

```diff
@@ -0,0 +1,8 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public enum DeathCause
+{
+    Hunger, Thirst, Age, Predators, Other
```

Comment:
```suggestion
    Other, Hunger, Thirst, Age, Predators
```
Den som står först blir default. Det kan vara bra om man råkar glömma att välja death cause när man skapar ett death event.